api/create_materialized_view.md

```diff
@@ -21,93 +21,85 @@ GROUP BY time_bucket( <const_value>, <partition_col_of_hypertable> ),
          [ optional grouping exprs>]
 [HAVING ...]
 ```
-Note that continuous aggregates have some limitations of what types of
-queries they can support, described in more length below.  For example,
-the `FROM` clause must provide only one hypertable, i.e., no joins, CTEs, views or
-subqueries are supported. The `GROUP BY` clause must include a time bucket on
-the hypertable's time column, and all aggregates must be parallelizable.
+
+Continuous aggregates have some limitations of what types of queries they can
+support, described in more length below.  For example, the `FROM` clause must
+provide only one hypertable, and joins, CTEs, views or subqueries are not
+supported. The `GROUP BY` clause must include a time bucket on the hypertable
+time column, and all aggregates must be parallelizable.
+
+The continuous aggregate view is automatically refreshed unless `WITH NO DATA` is given. This setting defaults to `WITH DATA`. For more information, see [`refresh_continuous_aggregate`][refresh-cagg].
+
+Some important things to remember when constructing your `SELECT` query:
```

Comment:
Perhaps move the paragraph "The continuous aggregate" view to above the limitations paragraph above it?  Because the following section goes into more limitations, seems like to makes more sense to:

1. Mentioned the WITH NO DATA parameter.
2. Summarize and detail limitations.